backend/package.json

```
@@ -10,7 +10,7 @@
     "lint:type": "npx tsc --noEmit",
     "lint:js": "npx eslint \"src/**/*.ts\"",
     "start:dev": "nodemon --exec \"node --loader ts-paths-esm-loader\" src/index.ts",
-    "migrate:dev": "node --loader ts-paths-esm-loader ../node_modules/knex/bin/cli.js migrate:latest",
+    "migrate:dev": "node --watch --loader ts-paths-esm-loader ../node_modules/knex/bin/cli.js migrate:latest",
```

Don't know for what reason migration script should use `--watch`, and how it relates to ProtectedRoute on FE
```suggestion
    "migrate:dev": "node --loader ts-paths-esm-loader ../node_modules/knex/bin/cli.js migrate:latest",
```

---

"migrate:dev": "node --watch --loader ts-paths-esm-loader ../node_modules/knex/bin/cli.js migrate:latest",